src/components/footer/Footer.tsx

```diff
@@ -157,6 +158,9 @@ export function Footer(props: FooterProps) {
 								&nbsp;
 							</li>
 						))}
+						<li className={fr.cx('fr-footer__bottom-item')}>
+							<AppVersion className={fr.cx('fr-footer__bottom-link')} />
```

Comment:
Will the class `fr-footer__bottom-link` not make it look like a link?

Reply:
It will look like the other items, without the hover state.
![image(8)](https://github.com/InseeFr/Stromae/assets/5951368/fcdd2222-c4a8-4372-bff9-6fff5f2adab3)